src/scene/scene_lua.inl

```diff
@@ -1234,6 +1234,31 @@ enum class [[codegen::enum]] CustomPropertyType {
     return openspace::makeIdentifier(input);
 }
 
+/**
+ * Set a custom ordering of the items in a specific branch in the Scene GUI tree, i.e.
+ * for a specific GUI path.
+ *
+ * \param guiPath The GUI path for which the order should be set.
+ * \param list A list of names of scene graph nodes or subgroups in the GUI, in the order
+ *             of which they should appear in the tree. The list does not have to include
+ *             all items in the given GUI path. Any excluded items will be placed after
+ *             the ones in the list.
+ */
+[[codegen::luawrap]] void setGuiTreeOrdering(std::string guiPath,
```

Comment:
```suggestion
[[codegen::luawrap]] void setGuiSceneOrdering(std::string guiPath,
```

To ensure that it is the Scene menu?

Reply:
Good point.  It will only work in the Scene and only for `groups`, not properties within property owners, for example. The "gui tree" might imply the entire tree of propertyowners, etc. 

Should it maybe be even more specific?  I used the word "gui group" before chaning it to "gui tree". Maybe "setGuiSceneGroupOrdering" ? 

Need some discussion and input...

Reply:
I did end up removing the word "group" as I thought it would not be familiar to the user, for example. But maybe that doesn't really matter